packages/webapp/components/layouts/AccountLayout/Security/index.tsx

```diff
@@ -142,6 +143,7 @@ function AccountSecurityDefault({
       await deleteAccount();
       await onUpdateSignBack(null, null);
       window.location.replace('/');
+      globalThis?.localStorage.removeItem(BOOT_LOCAL_KEY);
```

Comment:
I think you might need to call it before the replace (it might become a race condition otherwise)